contrib/docker/Dockerfile.downstream

```diff
@@ -1,22 +1,40 @@
-#!/usr/bin/env bash
+ARG TARGETARCH
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:v1.22.5-202407301806.g4c8b32d.el9 as builder
 
-echo "Updating container file"
+ARG TARGETARCH=amd64
+WORKDIR /app
 
-: "${CONTAINER_FILE:=./contrib/docker/Dockerfile}"
-: "${COMMIT:=$(git rev-list --abbrev-commit --tags --max-count=1)}"
+# Copy source code
+COPY go.mod .
+COPY go.sum .
+COPY Makefile .
+COPY .mk/ .mk/
+COPY .bingo/ .bingo/
+COPY vendor/ vendor/
+COPY .git/ .git/
+COPY cmd/ cmd/
+COPY pkg/ pkg/
+
+RUN git status --porcelain
+RUN GOARCH=$TARGETARCH make build_code
+
+# final stage
+FROM --platform=linux/$TARGETARCH registry.access.redhat.com/ubi9/ubi-minimal:9.4
+
+COPY --from=builder /app/flowlogs-pipeline /app/
+COPY --from=builder /app/confgenerator /app/
```

Comment:
iirc in cpaas midstream we did not include the confgenerator, it was only used in the upstream image